include/amici/steadystateproblem.h

```diff
@@ -93,13 +93,32 @@ class SteadystateProblem {
                                      int it);
 
     /**
-     * Compute the quadrature in steady state backward mode by solving the
+     * Handles the computation of the quadrature in steady state backward mode
+     *
+     * @param newtonSolver pointer to the newtonSolver solver object
+     * @param solver pointer to the solver object
+     * @param model pointer to the model object
+     */
+    void computeSteadyStateQuadrature(NewtonSolver *newtonSolver,
```

Comment:
I split the old backward routine up to flatten out logic, similar to what was done in the forward part.